### DISCUSSION TITLE
How can I manage categories through admin UI

### DISCUSSION THREAD

**Discussion by mister-loop** [medusajs/medusa#4492] [Q&A]
I was looking at the backend and frontend of the Vercel commerce's medusa version. I plugged the admin dashboard to the backend and it works great, exept that I couldn't find many controls such as managing cateogries and adding producst onto those categories. 
The frontend uses the category endpoints and the seeds too. I looked at documentation and the widgets as well as the customization sections, but they are a little too advanced and overkill for my situation.
I just wanted to do crud operations on the categories and be able to add/delete products in them.

**Comment by srindom** [comment]
There should be a Categories section in your admin application where you can manage your categories and structure the category tree.

Additionally, you can add products to categories by going to the product detail page and adding them to the desired categories.

**Reply by mister-loop** [reply]
I am only able to see the collection section on both the places

**Reply by tsvetann** [reply]
The categories are under a flag which needs to be enabled

**Comment by thoughtAddict** [comment]
As of Aug 2023, if you're not seeing "Categories" in a fresh Medusa install Admin UI, you have to manually set the flag to "true" in your .env file, (or .env.local, .env.test, etc):

`MEDUSA_FF_PRODUCT_CATEGORIES=true`

After adding that flag, I found I had to restart the Medusa server:

`npx @medusajs/medusa-cli develop`

See the following for more info:

https://docs.medusajs.com/modules/products/categories

**Comment by faridghar** [comment]
Worth adding that, depending on when you created your medusa backend database, you may need to run `npx medusa migrations run
` in order to update your database schema before product categories start working correctly.
